clustoapi/apps/attribute.py

```diff
@@ -21,6 +21,61 @@
 app.config['source_module'] = __name__
 
 
+def _write_attrs(method, name, **kwargs):
+    """
+Helper method for reduced code between POST and PUT.
+Returns a response for the methods calling it.
+"""
+    if method == 'set_attr':
```

Comment:
The reason that is added is because https://github.com/clusto/clusto-apiserver/pull/22/files#diff-aa496ad3f22a84193a0445005d60c6b5R74